cves/kernel/CVE-2015-4178.yml

```diff
@@ -280,8 +284,8 @@ ipc:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: false
+  note: The issue was with a data structure not being safely initialized, no ipc was used.
```

Comment:
"IPC"

Reply:
Thanks for catching that, it has been corrected.